datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/metadata.py

```diff
@@ -276,8 +276,8 @@ def metadata(check, check_duplicates):
             if row['metric_name'] != normalized_metric_name:
                 errors = True
                 echo_failure(
-                    f"Metric name '{row['metric_name']}' is not valid,"
-                    "it should be normalized as {normalized_metric_name}"
+                    f"{current_check}:{line} Metric name '{row['metric_name']}' is not valid,"
```

Comment:
```suggestion
                    f"{current_check}:{line} Metric name '{row['metric_name']}' is not valid, "
```
nit: missing space after comma